src/components/App.tsx

```diff
@@ -682,6 +682,14 @@ class App extends React.Component<AppProps, AppState> {
         if (typeof this.props.name !== "undefined") {
           name = this.props.name;
         }
+
+        const fileName = actionResult?.appState?.fileHandle?.name;
+        if (fileName) {
+          document.title = fileName;
+        } else {
+          document.title = APP_NAME;
+        }
+
```

Comment:
```suggestion
        document.title = actionResult?.appState?.fileHandle?.name || APP_NAME;

```

Reply:
Cool, yes we can have it like this, thanks